packages/utils/theme-node/vite.config.ts

```diff
@@ -1,9 +1,10 @@
 import { terser } from 'rollup-plugin-terser'
 import dts from 'vite-plugin-dts'
+import browserslistToEsbuild from 'browserslist-to-esbuild'
 
 export default {
   build: {
-    target: 'es2015',
+    target: browserslistToEsbuild(),
```

Comment:
Does this make sense ? This a node package

Reply:
Since Vite does not support browserlist configuration, a transformer helper named marcofugaro/browserslist-to-esbuild is used that basically when called: browserslistToEsbuild() transforms "> 0.5% or > 0.1% and last 2 years, not dead, not Firefox ESR, not and_qq > 0, not and_uc > 0, not and_ff >0" to [ 'chrome98', 'edge107', 'firefox108', 'ios14.5', 'safari14.1' ]